src/components/main/components/card/Card.scss

```
@@ -0,0 +1,48 @@
+.card {
+    max-width: 360px;
+    max-height: 506px;
+    height: 100%;
+    margin-bottom: 80px;
+
+    img {
+        width: 100%;
+        height: 100%;
+        max-height: 360px;
+        object-fit: cover;
+    }
+
+    .card__description {
```

поправь по аналогии стили, пожалуйста

```suggestion
    &__description {
```